<launch>

  <!-- defines name of the namespace -->
  <arg name="UAV_NAME" default="$(optenv UAV_NAME uav1)" />

  <arg name="custom_config" default="" />

  <!-- will it run using GNU debugger? -->
  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix_debug" value=""/>
  <arg     if="$(arg debug)" name="launch_prefix_debug" value="debug_roslaunch"/>


  <group ns="$(arg UAV_NAME)">

    <node name="mission_handler_test_client" pkg="iroc_mission_handler" type="mission_handler_test_client" launch-prefix="$(arg launch_prefix_debug)" output="screen">

      <!-- action server remap -->
      <remap from="waypoint_mission_action" to="mission_handler" />

    </node>

  </group>

</launch>
